### PR TITLE
Improve json_request.py coverage

### PR DESCRIPTION
**Pull request by noon-io**
Improving branch coverage within json_request.py

### CODE REVIEW

File: tests/test_http_request.py [hunk 1]
```diff
@@ -1642,6 +1642,26 @@ def test_replace_dumps_kwargs(self):
...
+        body2 = b"body"
+        data1 = {
+            "name1": "value1",
+        }
+        data2 = {
+            "name2": "value2",
+        }
+        r1 = self.request_class(url="http://www.example.com/", data=data1, body=body1)
+
+        with mock.patch("warnings.warn") as mock_warn:
```
Comment (Gallaecio) [inline tests/test_http_request.py:1657]: https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html

File: tests/test_http_request.py [hunk 2]
```diff
@@ -1642,6 +1642,26 @@ def test_replace_dumps_kwargs(self):
             self.assertEqual(kwargs["ensure_ascii"], True)
             self.assertEqual(kwargs["allow_nan"], True)
 
+    def test_replacement_both_body_and_data_warns(self):
+        """Test that we can get a warning if both body and data are passed for branch coverage"""
```
Comment (wRAR) [inline tests/test_http_request.py:1646]: Can you please remove the branch coverage mentions from here and other similar changes in other PRs? I don't think they are useful.

**Comment by noon-io** [comment]
@Gallaecio @wRAR Thanks for the feedback! I hope that the issues raised in the reviews have been addressed.

**Review by Gallaecio — APPROVED** [review]

**Review by wRAR — APPROVED** [review]

**Comment by wRAR** [comment]
Thanks!

**Pull request merged**